docs/_includes/skeleton-loader.html

```diff
@@ -0,0 +1,384 @@
+<div id="skeleton-loader">
+  <!-- {% include section-header.html name="skeleton-loader" version=page.versions.skeleton-loader %} -->
+  <h2>@ebay/skin/skeleton-loader <sup class="beta">BETA</sup></h2>
+  <p>The Skeletons are simplified versions of layouts to indicate that information has not
+    been fully loaded.</p>
+  <ul>
+    <li>Skeletons should only be used for loads that take 500ms or longer.</li>
```

Comment:
```suggestion
    <li>Skeletons should only appear when loading time reaches 500ms or longer.</li>
```

Reply:
Actually, I'll hold back on doc suggestion changes until we decide on final name ("skeleton", "skeleton-loader", "loading", etc) and all files are renamed.